diff --git a/data/org.gnome.Calculator.appdata.xml.in b/data/org.gnome.Calculator.appdata.xml.in
index e981d537..5f318c72 100644
--- a/data/org.gnome.Calculator.appdata.xml.in
+++ b/data/org.gnome.Calculator.appdata.xml.in
@@ -82,7 +82,7 @@
         </ul>
       </description>
     </release>
-    <release version="41.rc" date="2021-09-04" type="development">
+    <release version="41~rc" date="2021-09-04" type="development">
       <description>
         <p>Overview of changes in gnome-calculator 41.rc</p>
         <ul>
@@ -90,7 +90,7 @@
         </ul>
       </description>
     </release>
-    <release version="41.alpha" date="2021-07-10" type="development">
+    <release version="41~alpha" date="2021-07-10" type="development">
       <description>
         <p>Overview of changes in gnome-calculator 41.alpha</p>
         <ul>
@@ -115,7 +115,7 @@
         </ul>
       </description>
     </release>
-    <release version="40.rc" date="2021-03-12" type="development">
+    <release version="40~rc" date="2021-03-12" type="development">
       <description>
         <p>Overview of changes in gnome-calculator 40.rc</p>
         <ul>
@@ -133,7 +133,7 @@
         </ul>
       </description>
     </release>  
-    <release version="40.beta" date="2021-02-05" type="development">
+    <release version="40~beta" date="2021-02-05" type="development">
       <description>
         <p>Overview of changes in gnome-calculator 40.beta</p>
         <ul>
@@ -153,7 +153,7 @@
         </ul>
       </description>
     </release>  
-    <release version="40.alpha" date="2021-01-10" type="development">
+    <release version="40~alpha" date="2021-01-10" type="development">
       <description>
         <p>Overview of changes in gnome-calculator 40.alpha</p>
         <ul>
